polymesh-docs/primitives/nfts.mdx

```diff
@@ -46,4 +46,86 @@ Once the dispatchable succeeds, the non-fungible token will no longer be linked
+* if the individual NFT has a `tokenURI` value set, use that
+* else if the NFT collection has a `baseTokenURI` value set then:
+  * if the `baseTokenURI` does not already include `{tokenId}` append `/{tokenId}` and use that (e.g. the resolved URI is {baseTokenUri}/{tokenId})
+  * otherwise use the unmodified `baseTokenURI`
+* substitute any `{tokenId}` references with the underlying `tokenId` of the individual NFT
+
+The JSON blob referenced by an individual NFTs Token URI should confirm to the JSON schema below:
+```json
+{
+    "title": "Token Metadata",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string",
+            "description": "Identifies the asset which this token represents"
+        },
+        "description": {
+            "type": "string",
+            "description": "Describes the asset which this token represents"
+        },
+        "image": {
+            "type": "string",
+            "description": "A URI pointing to a resource with mime type image/* representing the asset to which this token represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive."
+        },
+        "properties": {
+            "type": "object",
+            "description": "Arbitrary properties. Values may be strings, numbers, object or arrays."
+        }
+    }
+}
+```
+
+### Image URI
+
+As well as being able to specify off-chain metadata via a JSON blob, you can also directly specify the image associated with an individual NFT.
+
+The Image URI of an NFT is intended to point towards an off-chain URI containing image data for the individual NFT.
+
+The Image URI can be specified in various ways:
+* via the global metadata key `imageURI` defined for an individual NFT
+* via the global metadata key `baseImageURI` defined for the whole NFT collection
+
+In both cases, the metadata value can include `{tokenId}` which will be substituted by a client with the corresponding `tokenId` of the individual NFT being referenced.
+
+The full process to determine the Image URI of an individual NFT is:
+* if the individual NFT has a `imageURI` value set, use that
+* else if the NFT collection has a `baseImageURI` value set then:
+  * if the `baseImageURI` does not already include `{tokenId}` append `/{tokenId}` and use that (e.g. the resolved URI is {baseImageURI}/{tokenId})
+  * otherwise use the unmodified `baseImageURI`
+* substitute any `{tokenId}` references with the underlying `tokenId` of the individual NFT
+
+This logic is equivalent to the above logic for [Token URI](#token-uri).
+
+:::Note
+
+When using `baseImageURI`, it is highly recommended to include `{tokenId}` to ensure a file extension is included in the URI. Without the file extension, there may be ambiguity in how the image is handled, which can lead to display issues or incorrect handling by applications reading the file.
```

Comment:
I think applications figure out the file type either through a header, be it file or http.

> `{tokenId}` to ensure a file extension is included in the URI.

the template allows for this to happen, but it doesn't ensure it.

I think we can remove this Note myself, since it a nice thing to do, not that application will get confused